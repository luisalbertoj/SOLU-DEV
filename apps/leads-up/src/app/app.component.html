<input type="text" [(ngModel)]="message.text" name="message.text" />
<button (click)="sendMessage()">send</button>
<a routerLink="/">Home</a>
<a routerLink="/auth/login">Login</a>
<a routerLink="/auth/register">Register</a>

<router-outlet></router-outlet>
