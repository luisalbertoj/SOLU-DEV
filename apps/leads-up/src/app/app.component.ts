import { Component } from '@angular/core';
import { HttpClient } from '@angular/common/http';
import { Message } from '@solu-dev/api-interfaces';

@Component({
  selector: 'solu-dev-root',
  templateUrl: './app.component.html',
  styleUrls: ['./app.component.css'],
})
export class AppComponent {
  hello$ = this.http.get<Message>('/api/hello');
  constructor(private http: HttpClient) {}
}
