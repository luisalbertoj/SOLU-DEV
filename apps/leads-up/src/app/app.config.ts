import { Injectable } from '@angular/core';
import { JwtHelperService } from '@auth0/angular-jwt';

const helper = new JwtHelperService();

@Injectable()
export class AppConfig {
  load() {
    return new Promise((resolve) => {
      const token =
        localStorage.getItem('auth_token') === null
          ? false
          : localStorage.getItem('auth_token');
      if (token) {
        try {
          if (helper.isTokenExpired(token) === true) {
            localStorage.removeItem('auth_token');
          }
        } catch (exception) {
          localStorage.removeItem('auth_token');
        }
      }
      resolve(true);
    });
  }
}
