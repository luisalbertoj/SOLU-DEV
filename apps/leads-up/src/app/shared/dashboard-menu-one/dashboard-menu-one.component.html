<div
  class="layout-wrapper"
  (click)="onLayoutClick()"
  [ngClass]="{
    'layout-static': app.menuMode === 'static',
    'layout-overlay': app.menuMode === 'overlay',
    'layout-slim': app.menuMode === 'slim',
    'layout-horizontal': app.menuMode === 'horizontal',
    'layout-sidebar': app.menuMode === 'sidebar',
    'layout-sidebar-static': app.menuMode === 'sidebar' && sidebarStatic,
    'layout-overlay-active': overlayMenuActive,
    'layout-mobile-active': staticMenuMobileActive,
    'layout-static-inactive':
      staticMenuDesktopInactive && app.menuMode === 'static',
    'p-ripple-disabled': !app.ripple
  }"
>
  <div class="layout-main">
    <solu-dev-top-bar-one
      *ngIf="isMobile() || isOverlay() || isHorizontal()"
    ></solu-dev-top-bar-one>

    <solu-dev-menu-one
      *ngIf="!isMobile() && (isStatic() || isSlim() || isSidebar())"
    ></solu-dev-menu-one>

    <solu-dev-breadcrumb-one></solu-dev-breadcrumb-one>

    <div class="layout-main-content">
      <router-outlet></router-outlet>
    </div>

    <solu-dev-footer-one></solu-dev-footer-one>
  </div>

  <solu-dev-right-menu-one></solu-dev-right-menu-one>

  <solu-dev-app-config></solu-dev-app-config>
</div>
