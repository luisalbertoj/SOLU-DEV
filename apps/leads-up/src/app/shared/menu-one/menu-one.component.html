<div
  class="layout-menu-wrapper"
  [ngClass]="{ 'layout-sidebar-active': appMain.sidebarActive$ | async }"
  (click)="appMain.onSidebarClick()"
  (mouseover)="appMain.onSidebarMouseOver()"
  (mouseleave)="appMain.onSidebarMouseLeave()"
>
  <div class="menu-logo" *ngIf="!appMain.isHorizontal() || appMain.isMobile()">
    <a href="#" class="logo">
      <img
        [src]="
          'assets/layout/images/logo-' +
          (app.colorScheme === 'light' ? 'dark' : 'light') +
          '.png'
        "
      />
    </a>

    <a
      href="#"
      class="app-name"
      *ngIf="!appMain.isSlim() || appMain.isMobile()"
    >
      <img
        [src]="
          'assets/layout/images/appname-' +
          (app.colorScheme === 'light' ? 'dark' : 'light') +
          '.png'
        "
      />
    </a>
    <a href="#" class="menu-pin" (click)="appMain.onToggleMenu($event)">
      <span *ngIf="appMain.isOverlay()" class="pi pi-times"></span>
      <span
        *ngIf="
          appMain.isSidebar() && !appMain.sidebarStatic && appMain.pinActive
        "
        class="pi pi-unlock"
      ></span>
      <span
        *ngIf="
          appMain.isSidebar() && appMain.sidebarStatic && appMain.pinActive
        "
        class="pi pi-lock"
      ></span>
    </a>
  </div>

  <div class="layout-menu-container">
    <ul class="layout-menu">
      <ng-container *ngFor="let item of model; let i = index">
        <li
          solu-dev-menu-item
          *ngIf="!item.separator"
          [item]="item"
          [index]="i"
          [root]="true"
        ></li>
        <li *ngIf="item.separator" class="menu-separator"></li>
      </ng-container>
    </ul>
  </div>

  <solu-dev-inline-menu-one></solu-dev-inline-menu-one>
</div>
