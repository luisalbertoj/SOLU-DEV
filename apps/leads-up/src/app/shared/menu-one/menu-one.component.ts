import { CommonModule } from '@angular/common';
import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
import { AppComponent } from '../../app.component';
import { DashboardMenuOneComponent } from '../dashboard-menu-one/dashboard-menu-one.component';
import { InlineMenuOneComponent } from '../inline-menu-one/inline-menu-one.component';
import { MenuItemComponent } from '../menu-item/menu-item.component';

@Component({
  selector: 'solu-dev-menu-one',
  standalone: true,
  imports: [CommonModule, MenuItemComponent, InlineMenuOneComponent],
  templateUrl: './menu-one.component.html',
  styleUrls: ['./menu-one.component.scss'],
  changeDetection: ChangeDetectionStrategy.OnPush,
})
export class MenuOneComponent implements OnInit {
  // eslint-disable-next-line @typescript-eslint/no-explicit-any
  public model!: any[];

  constructor(
    public app: AppComponent,
    public appMain: DashboardMenuOneComponent
  ) {}

  ngOnInit() {
    this.model = [
      {
        label: 'Favorites',
        icon: 'pi pi-home',
        items: [
          { label: 'Dashboard', icon: 'pi pi-fw pi-home', routerLink: ['/'] },
        ],
      },
      {
        label: 'UI Kit',
        icon: 'pi pi-fw pi-star',
        routerLink: ['/uikit'],
        items: [
          {
            label: 'Form Layout',
            icon: 'pi pi-fw pi-id-card',
            routerLink: ['/uikit/formlayout'],
          },
          {
            label: 'Input',
            icon: 'pi pi-fw pi-check-square',
            routerLink: ['/uikit/input'],
          },
          {
            label: 'Float Label',
            icon: 'pi pi-fw pi-bookmark',
            routerLink: ['/uikit/floatlabel'],
          },
          {
            label: 'Invalid State',
            icon: 'pi pi-fw pi-exclamation-circle',
            routerLink: ['/uikit/invalidstate'],
          },
          {
            label: 'Button',
            icon: 'pi pi-fw pi-mobile',
            routerLink: ['/uikit/button'],
            class: 'rotated-icon',
          },
          {
            label: 'Table',
            icon: 'pi pi-fw pi-table',
            routerLink: ['/uikit/table'],
          },
          {
            label: 'List',
            icon: 'pi pi-fw pi-list',
            routerLink: ['/uikit/list'],
          },
          {
            label: 'Tree',
            icon: 'pi pi-fw pi-share-alt',
            routerLink: ['/uikit/tree'],
          },
          {
            label: 'Panel',
            icon: 'pi pi-fw pi-tablet',
            routerLink: ['/uikit/panel'],
          },
          {
            label: 'Overlay',
            icon: 'pi pi-fw pi-clone',
            routerLink: ['/uikit/overlay'],
          },
          {
            label: 'Media',
            icon: 'pi pi-fw pi-image',
            routerLink: ['/uikit/media'],
          },
          {
            label: 'Menu',
            icon: 'pi pi-fw pi-bars',
            routerLink: ['/uikit/menu'],
            preventExact: true,
          },
          {
            label: 'Message',
            icon: 'pi pi-fw pi-comment',
            routerLink: ['/uikit/message'],
          },
          {
            label: 'File',
            icon: 'pi pi-fw pi-file',
            routerLink: ['/uikit/file'],
          },
          {
            label: 'Chart',
            icon: 'pi pi-fw pi-chart-bar',
            routerLink: ['/uikit/charts'],
          },
          {
            label: 'Misc',
            icon: 'pi pi-fw pi-circle-off',
            routerLink: ['/uikit/misc'],
          },
        ],
      },
      {
        label: 'Utilities',
        icon: 'pi pi-fw pi-compass',
        routerLink: ['/utilities'],
        items: [
          {
            label: 'PrimeIcons',
            icon: 'pi pi-fw pi-prime',
            routerLink: ['utilities/icons'],
          },
          {
            label: 'PrimeFlex',
            icon: 'pi pi-fw pi-desktop',
            url: ['https://www.primefaces.org/primeflex/'],
            target: '_blank',
          },
        ],
      },
      {
        label: 'Pages',
        icon: 'pi pi-fw pi-briefcase',
        routerLink: ['/pages'],
        items: [
          {
            label: 'Crud',
            icon: 'pi pi-fw pi-pencil',
            routerLink: ['/pages/crud'],
          },
          {
            label: 'Calendar',
            icon: 'pi pi-fw pi-calendar-plus',
            routerLink: ['/pages/calendar'],
          },
          {
            label: 'Timeline',
            icon: 'pi pi-fw pi-calendar',
            routerLink: ['/pages/timeline'],
          },
          {
            label: 'Landing',
            icon: 'pi pi-fw pi-globe',
            url: 'assets/pages/landing.html',
            target: '_blank',
          },
          {
            label: 'Login',
            icon: 'pi pi-fw pi-sign-in',
            routerLink: ['/login'],
          },
          {
            label: 'Invoice',
            icon: 'pi pi-fw pi-dollar',
            routerLink: ['/pages/invoice'],
          },
          {
            label: 'Help',
            icon: 'pi pi-fw pi-question-circle',
            routerLink: ['/pages/help'],
          },
          {
            label: 'Error',
            icon: 'pi pi-fw pi-times-circle',
            routerLink: ['/error'],
          },
          {
            label: 'Not Found',
            icon: 'pi pi-fw pi-exclamation-circle',
            routerLink: ['/notfound'],
          },
          {
            label: 'Access Denied',
            icon: 'pi pi-fw pi-lock',
            routerLink: ['/access'],
          },
          {
            label: 'Empty',
            icon: 'pi pi-fw pi-circle-off',
            routerLink: ['/pages/empty'],
          },
        ],
      },
      {
        label: 'Hierarchy',
        icon: 'pi pi-fw pi-align-left',
        items: [
          {
            label: 'Submenu 1',
            icon: 'pi pi-fw pi-align-left',
            items: [
              {
                label: 'Submenu 1.1',
                icon: 'pi pi-fw pi-align-left',
                items: [
                  { label: 'Submenu 1.1.1', icon: 'pi pi-fw pi-align-left' },
                  { label: 'Submenu 1.1.2', icon: 'pi pi-fw pi-align-left' },
                  { label: 'Submenu 1.1.3', icon: 'pi pi-fw pi-align-left' },
                ],
              },
              {
                label: 'Submenu 1.2',
                icon: 'pi pi-fw pi-align-left',
                items: [
                  { label: 'Submenu 1.2.1', icon: 'pi pi-fw pi-align-left' },
                ],
              },
            ],
          },
          {
            label: 'Submenu 2',
            icon: 'pi pi-fw pi-align-left',
            items: [
              {
                label: 'Submenu 2.1',
                icon: 'pi pi-fw pi-align-left',
                items: [
                  { label: 'Submenu 2.1.1', icon: 'pi pi-fw pi-align-left' },
                  { label: 'Submenu 2.1.2', icon: 'pi pi-fw pi-align-left' },
                ],
              },
              {
                label: 'Submenu 2.2',
                icon: 'pi pi-fw pi-align-left',
                items: [
                  { label: 'Submenu 2.2.1', icon: 'pi pi-fw pi-align-left' },
                ],
              },
            ],
          },
        ],
      },
      {
        label: 'Start',
        icon: 'pi pi-fw pi-download',
        items: [
          {
            label: 'Buy Now',
            icon: 'pi pi-fw pi-shopping-cart',
            url: ['https://www.primefaces.org/store'],
          },
          {
            label: 'Documentation',
            icon: 'pi pi-fw pi-info-circle',
            routerLink: ['/documentation'],
          },
        ],
      },
    ];
  }
}
