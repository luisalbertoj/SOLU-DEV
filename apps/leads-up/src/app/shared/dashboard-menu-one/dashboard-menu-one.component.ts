import { CommonModule } from '@angular/common';
import { Component } from '@angular/core';
import { RouterModule } from '@angular/router';
import { PrimeNGConfig } from 'primeng/api';
import { Subject } from 'rxjs';
import { AppComponent } from '../../app.component';
import { AppConfigComponent } from '../app-config/app-config.component';
import { BreadcrumbOneComponent } from '../breadcrumb-one/breadcrumb-one.component';
import { FooterOneComponent } from '../footer-one/footer-one.component';
import { MenuOneComponent } from '../menu-one/menu-one.component';
import { RightMenuOneComponent } from '../right-menu-one/right-menu-one.component';
import { TopBarOneComponent } from '../top-bar-one/top-bar-one.component';
import { MenuService } from './services/dashboard-menu.service';

@Component({
  selector: 'solu-dev-dashboard-menu-one',
  standalone: true,
  imports: [
    CommonModule,
    TopBarOneComponent,
    MenuOneComponent,
    BreadcrumbOneComponent,
    RouterModule,
    AppConfigComponent,
    FooterOneComponent,
    RightMenuOneComponent,
  ],
  providers: [MenuService],
  templateUrl: './dashboard-menu-one.component.html',
  styleUrls: ['./dashboard-menu-one.component.scss'],
})
export class DashboardMenuOneComponent {
  overlayMenuActive!: boolean;

  staticMenuDesktopInactive = false;

  staticMenuMobileActive!: boolean;

  sidebarActive$ = new Subject<boolean>();

  sidebarActive = false;

  sidebarStatic = false;

  menuClick!: boolean;

  menuHoverActive = false;

  topbarMenuActive!: boolean;

  topbarItemClick!: boolean;

  // eslint-disable-next-line @typescript-eslint/no-explicit-any
  activeTopbarItem!: any;

  configActive$ = new Subject<boolean>();

  configActive!: boolean;

  configClick!: boolean;

  rightMenuActive$ = new Subject<boolean>();

  rightMenuActive!: boolean;

  rightMenuClick!: boolean;

  searchActive!: boolean;

  searchClick!: boolean;

  activeInlineProfile!: boolean;

  pinActive!: boolean;

  constructor(
    private menuService: MenuService,
    private primengConfig: PrimeNGConfig,
    public app: AppComponent
  ) {}

  onLayoutClick() {
    if (!this.topbarItemClick) {
      this.activeTopbarItem = null;
      this.topbarMenuActive = false;
    }

    if (this.configActive && !this.configClick) {
      this.configActive = false;
      this.configActive$.next(false);
    }

    if (this.rightMenuActive && !this.rightMenuClick) {
      this.rightMenuActive$.next(false);
      this.rightMenuActive = false;
    }

    if (this.searchActive && !this.searchClick) {
      this.searchActive = false;
    }

    if (!this.menuClick) {
      if ((this.isSlim() || this.isHorizontal()) && !this.isMobile()) {
        this.menuService.reset();
        this.menuHoverActive = false;
      }

      if (this.overlayMenuActive || this.staticMenuMobileActive) {
        this.overlayMenuActive = false;
        this.staticMenuMobileActive = false;
      }
    }

    this.configClick = false;
    this.rightMenuClick = false;
    this.searchClick = false;
    this.menuClick = false;
    this.topbarItemClick = false;
  }

  onSidebarClick() {
    this.menuClick = true;
  }

  // eslint-disable-next-line @typescript-eslint/no-explicit-any
  onToggleMenu(event: any) {
    this.menuClick = true;

    if (this.overlayMenuActive) {
      this.overlayMenuActive = false;
    }

    if (this.sidebarActive) {
      this.sidebarStatic = !this.sidebarStatic;
    }

    event.preventDefault();
  }

  onSidebarMouseOver() {
    if (this.app.menuMode === 'sidebar' && !this.sidebarStatic) {
      this.sidebarActive$.next(this.isDesktop());
      this.sidebarActive = this.isDesktop();
      setTimeout(() => {
        this.pinActive = this.isDesktop();
      }, 200);
    }
  }

  onSidebarMouseLeave() {
    if (this.app.menuMode === 'sidebar' && !this.sidebarStatic) {
      setTimeout(() => {
        this.sidebarActive$.next(false);
        this.sidebarActive = false;
        this.pinActive = false;
      }, 250);
    }
  }
  // eslint-disable-next-line @typescript-eslint/no-explicit-any
  onMenuButtonClick(event: any) {
    this.menuClick = true;

    if (this.isOverlay()) {
      this.overlayMenuActive = !this.overlayMenuActive;
    }

    if (this.isDesktop()) {
      this.staticMenuDesktopInactive = !this.staticMenuDesktopInactive;
    } else {
      this.staticMenuMobileActive = !this.staticMenuMobileActive;
    }

    event.preventDefault();
  }
  // eslint-disable-next-line @typescript-eslint/no-explicit-any
  onTopbarItemClick(event: any, item: any) {
    this.topbarItemClick = true;

    if (this.activeTopbarItem === item) {
      this.activeTopbarItem = null;
    } else {
      this.activeTopbarItem = item;
    }

    event.preventDefault();
  }
  // eslint-disable-next-line @typescript-eslint/no-explicit-any
  onTopbarSubItemClick(event: any) {
    event.preventDefault();
  }
  // eslint-disable-next-line @typescript-eslint/no-explicit-any
  onRippleChange(event: any) {
    this.app.ripple = event.checked;
    this.primengConfig.ripple = event.checked;
  }

  onConfigClick() {
    this.configClick = true;
  }

  onRightMenuButtonClick() {
    this.rightMenuClick = true;
    this.rightMenuActive = true;
    this.rightMenuActive$.next(true);
  }

  onRightMenuClick() {
    this.rightMenuClick = true;
  }

  isStatic() {
    return this.app.menuMode === 'static';
  }

  isOverlay() {
    return this.app.menuMode === 'overlay';
  }

  isSlim() {
    return this.app.menuMode === 'slim';
  }

  isHorizontal() {
    return this.app.menuMode === 'horizontal';
  }

  isSidebar() {
    return this.app.menuMode === 'sidebar';
  }

  isDesktop() {
    return window.innerWidth > 991;
  }

  isMobile() {
    return window.innerWidth <= 991;
  }
}
