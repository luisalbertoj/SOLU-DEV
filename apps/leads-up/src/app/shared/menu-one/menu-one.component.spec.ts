import { ComponentFixture, TestBed } from '@angular/core/testing';
import { NoopAnimationsModule } from '@angular/platform-browser/animations';
import { RouterTestingModule } from '@angular/router/testing';
import { AppComponent } from '../../app.component';
import { DashboardMenuOneComponent } from '../dashboard-menu-one/dashboard-menu-one.component';
import { MenuService } from '../dashboard-menu-one/services/dashboard-menu.service';

import { MenuOneComponent } from './menu-one.component';

describe('MenuOneComponent', () => {
  let component: MenuOneComponent;
  let fixture: ComponentFixture<MenuOneComponent>;
  let compiled: HTMLElement;

  beforeEach(async () => {
    await TestBed.configureTestingModule({
      declarations: [],
      imports: [RouterTestingModule, NoopAnimationsModule],
      providers: [AppComponent, DashboardMenuOneComponent, MenuService],
    }).compileComponents();
  });

  beforeEach(() => {
    fixture = TestBed.createComponent(MenuOneComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
    compiled = fixture.nativeElement;
  });

  it('should create', () => {
    expect(component).toBeTruthy();
  });

  it('this should be equal to the snapshot', () => {
    expect(compiled.innerHTML).toMatchSnapshot();
  });
});
