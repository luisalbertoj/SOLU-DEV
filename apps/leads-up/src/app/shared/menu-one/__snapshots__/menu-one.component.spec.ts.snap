// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuOneComponent this should be equal to the snapshot 1`] = `
"<div class=\\"layout-menu-wrapper\\" ng-reflect-ng-class=\\"[object Object]\\"><div class=\\"menu-logo ng-star-inserted\\"><a href=\\"#\\" class=\\"logo\\"><img src=\\"assets/layout/images/logo-light.png\\"></a><a href=\\"#\\" class=\\"app-name ng-star-inserted\\"><img src=\\"assets/layout/images/appname-light.png\\"></a><!--bindings={
  \\"ng-reflect-ng-if\\": \\"true\\"
}--><a href=\\"#\\" class=\\"menu-pin\\"><!--bindings={
  \\"ng-reflect-ng-if\\": \\"false\\"
}--><!--bindings={}--><!--bindings={
  \\"ng-reflect-ng-if\\": \\"false\\"
}--></a></div><!--bindings={
  \\"ng-reflect-ng-if\\": \\"true\\"
}--><div class=\\"layout-menu-container\\"><ul class=\\"layout-menu\\"><li solu-dev-menu-item=\\"\\" class=\\"ng-tns-c62-1 layout-root-menuitem ng-star-inserted\\" ng-reflect-item=\\"[object Object]\\" ng-reflect-index=\\"0\\" ng-reflect-root=\\"true\\"><div class=\\"layout-menuitem-root-text ng-tns-c62-1 ng-star-inserted\\"> Favorites </div><!--bindings={
  \\"ng-reflect-ng-if\\": \\"true\\"
}--><a pripple=\\"\\" class=\\"p-ripple p-element ng-tns-c62-1 ng-star-inserted\\" tabindex=\\"0\\"><i class=\\"layout-menuitem-icon ng-tns-c62-1 pi pi-home\\" ng-reflect-ng-class=\\"pi pi-home\\"></i><span class=\\"layout-menuitem-text ng-tns-c62-1\\">Favorites</span><i class=\\"pi pi-fw pi-angle-down layout-submenu-toggler ng-tns-c62-1 ng-star-inserted\\"></i><!--bindings={
  \\"ng-reflect-ng-if\\": \\"[object Object]\\"
}--><!--bindings={}--><span class=\\"p-ink\\"></span></a><!--bindings={
  \\"ng-reflect-ng-if\\": \\"true\\"
}--><!--bindings={}--><!--bindings={
  \\"ng-reflect-ng-if\\": \\"false\\"
}--><!--ng-container--></li><!--bindings={
  \\"ng-reflect-ng-if\\": \\"true\\"
}--><!--bindings={}--><!--ng-container--><li solu-dev-menu-item=\\"\\" class=\\"ng-tns-c62-2 layout-root-menuitem ng-star-inserted\\" ng-reflect-item=\\"[object Object]\\" ng-reflect-index=\\"1\\" ng-reflect-root=\\"true\\"><div class=\\"layout-menuitem-root-text ng-tns-c62-2 ng-star-inserted\\"> UI Kit </div><!--bindings={
  \\"ng-reflect-ng-if\\": \\"true\\"
}--><a pripple=\\"\\" class=\\"p-ripple p-element ng-tns-c62-2 ng-star-inserted\\" tabindex=\\"0\\"><i class=\\"layout-menuitem-icon ng-tns-c62-2 pi pi-fw pi-star\\" ng-reflect-ng-class=\\"pi pi-fw pi-star\\"></i><span class=\\"layout-menuitem-text ng-tns-c62-2\\">UI Kit</span><i class=\\"pi pi-fw pi-angle-down layout-submenu-toggler ng-tns-c62-2 ng-star-inserted\\"></i><!--bindings={
  \\"ng-reflect-ng-if\\": \\"[object Object],[object Object\\"
}--><!--bindings={}--><span class=\\"p-ink\\"></span></a><!--bindings={
  \\"ng-reflect-ng-if\\": \\"true\\"
}--><!--bindings={
  \\"ng-reflect-ng-if\\": \\"false\\"
}--><!--bindings={
  \\"ng-reflect-ng-if\\": \\"false\\"
}--><!--ng-container--></li><!--bindings={
  \\"ng-reflect-ng-if\\": \\"true\\"
}--><!--bindings={}--><!--ng-container--><li solu-dev-menu-item=\\"\\" class=\\"ng-tns-c62-3 layout-root-menuitem ng-star-inserted\\" ng-reflect-item=\\"[object Object]\\" ng-reflect-index=\\"2\\" ng-reflect-root=\\"true\\"><div class=\\"layout-menuitem-root-text ng-tns-c62-3 ng-star-inserted\\"> Utilities </div><!--bindings={
  \\"ng-reflect-ng-if\\": \\"true\\"
}--><a pripple=\\"\\" class=\\"p-ripple p-element ng-tns-c62-3 ng-star-inserted\\" tabindex=\\"0\\"><i class=\\"layout-menuitem-icon ng-tns-c62-3 pi pi-fw pi-compass\\" ng-reflect-ng-class=\\"pi pi-fw pi-compass\\"></i><span class=\\"layout-menuitem-text ng-tns-c62-3\\">Utilities</span><i class=\\"pi pi-fw pi-angle-down layout-submenu-toggler ng-tns-c62-3 ng-star-inserted\\"></i><!--bindings={
  \\"ng-reflect-ng-if\\": \\"[object Object],[object Object\\"
}--><!--bindings={}--><span class=\\"p-ink\\"></span></a><!--bindings={
  \\"ng-reflect-ng-if\\": \\"true\\"
}--><!--bindings={
  \\"ng-reflect-ng-if\\": \\"false\\"
}--><!--bindings={
  \\"ng-reflect-ng-if\\": \\"false\\"
}--><!--ng-container--></li><!--bindings={
  \\"ng-reflect-ng-if\\": \\"true\\"
}--><!--bindings={}--><!--ng-container--><li solu-dev-menu-item=\\"\\" class=\\"ng-tns-c62-4 layout-root-menuitem ng-star-inserted\\" ng-reflect-item=\\"[object Object]\\" ng-reflect-index=\\"3\\" ng-reflect-root=\\"true\\"><div class=\\"layout-menuitem-root-text ng-tns-c62-4 ng-star-inserted\\"> Pages </div><!--bindings={
  \\"ng-reflect-ng-if\\": \\"true\\"
}--><a pripple=\\"\\" class=\\"p-ripple p-element ng-tns-c62-4 ng-star-inserted\\" tabindex=\\"0\\"><i class=\\"layout-menuitem-icon ng-tns-c62-4 pi pi-fw pi-briefcase\\" ng-reflect-ng-class=\\"pi pi-fw pi-briefcase\\"></i><span class=\\"layout-menuitem-text ng-tns-c62-4\\">Pages</span><i class=\\"pi pi-fw pi-angle-down layout-submenu-toggler ng-tns-c62-4 ng-star-inserted\\"></i><!--bindings={
  \\"ng-reflect-ng-if\\": \\"[object Object],[object Object\\"
}--><!--bindings={}--><span class=\\"p-ink\\"></span></a><!--bindings={
  \\"ng-reflect-ng-if\\": \\"true\\"
}--><!--bindings={
  \\"ng-reflect-ng-if\\": \\"false\\"
}--><!--bindings={
  \\"ng-reflect-ng-if\\": \\"false\\"
}--><!--ng-container--></li><!--bindings={
  \\"ng-reflect-ng-if\\": \\"true\\"
}--><!--bindings={}--><!--ng-container--><li solu-dev-menu-item=\\"\\" class=\\"ng-tns-c62-5 layout-root-menuitem ng-star-inserted\\" ng-reflect-item=\\"[object Object]\\" ng-reflect-index=\\"4\\" ng-reflect-root=\\"true\\"><div class=\\"layout-menuitem-root-text ng-tns-c62-5 ng-star-inserted\\"> Hierarchy </div><!--bindings={
  \\"ng-reflect-ng-if\\": \\"true\\"
}--><a pripple=\\"\\" class=\\"p-ripple p-element ng-tns-c62-5 ng-star-inserted\\" tabindex=\\"0\\"><i class=\\"layout-menuitem-icon ng-tns-c62-5 pi pi-fw pi-align-left\\" ng-reflect-ng-class=\\"pi pi-fw pi-align-left\\"></i><span class=\\"layout-menuitem-text ng-tns-c62-5\\">Hierarchy</span><i class=\\"pi pi-fw pi-angle-down layout-submenu-toggler ng-tns-c62-5 ng-star-inserted\\"></i><!--bindings={
  \\"ng-reflect-ng-if\\": \\"[object Object],[object Object\\"
}--><!--bindings={}--><span class=\\"p-ink\\"></span></a><!--bindings={
  \\"ng-reflect-ng-if\\": \\"true\\"
}--><!--bindings={}--><!--bindings={
  \\"ng-reflect-ng-if\\": \\"false\\"
}--><!--ng-container--></li><!--bindings={
  \\"ng-reflect-ng-if\\": \\"true\\"
}--><!--bindings={}--><!--ng-container--><li solu-dev-menu-item=\\"\\" class=\\"ng-tns-c62-6 layout-root-menuitem ng-star-inserted\\" ng-reflect-item=\\"[object Object]\\" ng-reflect-index=\\"5\\" ng-reflect-root=\\"true\\"><div class=\\"layout-menuitem-root-text ng-tns-c62-6 ng-star-inserted\\"> Start </div><!--bindings={
  \\"ng-reflect-ng-if\\": \\"true\\"
}--><a pripple=\\"\\" class=\\"p-ripple p-element ng-tns-c62-6 ng-star-inserted\\" tabindex=\\"0\\"><i class=\\"layout-menuitem-icon ng-tns-c62-6 pi pi-fw pi-download\\" ng-reflect-ng-class=\\"pi pi-fw pi-download\\"></i><span class=\\"layout-menuitem-text ng-tns-c62-6\\">Start</span><i class=\\"pi pi-fw pi-angle-down layout-submenu-toggler ng-tns-c62-6 ng-star-inserted\\"></i><!--bindings={
  \\"ng-reflect-ng-if\\": \\"[object Object],[object Object\\"
}--><!--bindings={}--><span class=\\"p-ink\\"></span></a><!--bindings={
  \\"ng-reflect-ng-if\\": \\"true\\"
}--><!--bindings={}--><!--bindings={
  \\"ng-reflect-ng-if\\": \\"false\\"
}--><!--ng-container--></li><!--bindings={
  \\"ng-reflect-ng-if\\": \\"true\\"
}--><!--bindings={}--><!--ng-container--><!--bindings={
  \\"ng-reflect-ng-for-of\\": \\"[object Object],[object Object\\"
}--></ul></div><solu-dev-inline-menu-one class=\\"ng-tns-c63-0\\"><div class=\\"ng-tns-c63-0 layout-inline-menu ng-star-inserted\\" ng-reflect-ng-class=\\"[object Object]\\"><a class=\\"layout-inline-menu-action ng-tns-c63-0\\"><img src=\\"assets/layout/images/profile-image.png\\" alt=\\"avatar\\" style=\\"width: 44px; height: 44px;\\" class=\\"ng-tns-c63-0\\"><span class=\\"layout-inline-menu-text ng-tns-c63-0\\">User Test</span><i class=\\"layout-inline-menu-icon pi pi-angle-down ng-tns-c63-0\\"></i></a><ul class=\\"layout-inline-menu-action-panel ng-tns-c63-0 ng-trigger ng-trigger-inline\\" style=\\"height:0px;overflow:hidden;0:height;1:overflow;\\"><li class=\\"layout-inline-menu-action-item ng-tns-c63-0\\"><a routerlinkactive=\\"router-link-active\\" class=\\"ng-tns-c63-0\\" ng-reflect-router-link-active=\\"router-link-active\\" ng-reflect-router-link=\\"/logout\\" href=\\"/logout\\"><i class=\\"pi pi-power-off pi-fw ng-tns-c63-0\\"></i><span class=\\"ng-tns-c63-0\\">Logout</span></a></li><li class=\\"layout-inline-menu-action-item ng-tns-c63-0\\"><a class=\\"ng-tns-c63-0\\"><i class=\\"pi pi-cog pi-fw ng-tns-c63-0\\"></i><span class=\\"ng-tns-c63-0\\">Settings</span></a></li><li class=\\"layout-inline-menu-action-item ng-tns-c63-0\\"><a class=\\"ng-tns-c63-0\\"><i class=\\"pi pi-user pi-fw ng-tns-c63-0\\"></i><span class=\\"ng-tns-c63-0\\">Profile</span></a></li></ul></div><!--bindings={
  \\"ng-reflect-ng-if\\": \\"true\\"
}--></solu-dev-inline-menu-one></div>"
`;
